docs/CONTRIBUTING-zh.md

```diff
@@ -42,17 +44,17 @@
     * 不要提供短链接
 * 优先选择使用 "current" 链接代替有 "version" 链接：使用 `http://example.com/dir/book/current/` 比使用 `http://example.com/dir/book/v1.0.0/index.html` 更好。
 * 如果一个链接存在过期的证书/自签名证书/SSL问题的任何其他类型：
-  1. *replace it* ：如果可能的话，将其 *替换* 为对应的`http`(因为在移动设备上接受异常可能比较复杂)。
-  2. *leave it* ：如果没有http版本，但仍然可以通过https访问链接，则在浏览器中添加异常或忽略警告。
-  3. *remove it* ：上述以外删除掉它。
+    1. *replace it* ：如果可能的话，将其 *替换* 为对应的`http`(因为在移动设备上接受异常可能比较复杂)。
+    2. *leave it* ：如果没有`http`版本，但仍然可以通过`https`访问链接，则在浏览器中添加异常或忽略警告。
+    3. *remove it* ：上述以外删除掉它。
 * 如果一个链接以多种格式存在，请添加一个单独的链接，并注明每种格式。
 * 如果一个资源存在于Internet上的不同位置
     * 使用最权威来源的链接(意思是原始作者的网站比编辑的网站好，比第三方网站好)。
     * 如果它们链接到不同的版本，你认为这些版本差异很大，值得保留，那么添加一个单独的链接，并对每个版本做一个说明(参见[Issue #2353](https://github.com/EbookFoundation/free-programming-books/issues/2353)有助于格式化问题的讨论)。
 * 相较一个比较大的提交，我们更倾向于原子提交(通过添加/删除/修改进行一次提交)。在提交PR之前没有必要压缩你的提交。(我们永远不会执行这个规则，因为这只是维护人员的方便)。
 * 如果一本书比较旧，请在书名中注明出版日期。
-* 包含作者的名字或适当的名字。中文版本可以用 “等” 缩短作者列表。
-* 如果一本书还没有完成，并且仍在编写中，则需添加 “编写中” 符号，参见[下文](#in_process)所述。
+* 包含作者的名字或适当的名字。中文版本可以用 “`等`” (“`et al.`”) 缩短作者列表。
+* 如果一本书还没有完成，并且仍在编写中，则需添加 “`in process`” 符号，参见[下文](#in_process)所述。
```

Comment:
why untranslate this?

Reply:
Same reason like in greek doc (see https://github.com/EbookFoundation/free-programming-books/pull/6625#discussion_r791060297)

Target lines: https://github.com/EbookFoundation/free-programming-books/blob/f4269cd6e82d77fdaeac0068333e462742c12f0f/docs/CONTRIBUTING-zh.md#L145-L150